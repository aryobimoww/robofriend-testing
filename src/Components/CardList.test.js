import {shallow} from 'enzyme';
import React from 'react';
import CardList from './CardList';
it('expect to render card component', ()=>{
    const mockRobots = [
        {
            id:1,
            name:'jane',
            username:'janejane',
            email:'jane@gmail.com'
        }
    ]
    expect(shallow(<CardList robots={mockRobots} />).debug()).toMatchSnapshot();
})
