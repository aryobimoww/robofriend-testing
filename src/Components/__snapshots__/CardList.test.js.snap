// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render card component 1`] = `
"<div>
  <Card id={1} name=\\"jane\\" email=\\"jane@gmail.com\\" />
</div>"
`;
